CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ set(ENABLE_IMPLICIT_CONVERSIONS ${CARE_ENABLE_IMPLICIT_CONVERSIONS} CACHE BOOL "
 option(CARE_LEGACY_COMPATIBILITY_MODE "Enable legacy compatibility mode" OFF)
 option(CARE_ENABLE_MANAGED_PTR "Enable managed_ptr aliases, tests, and reproducer" ON)
 option(CARE_DISABLE_RAJAPLUGIN "Disable use of the RAJA plugin. WILL ALSO DISABLE MEMORY MOTION." OFF)
+option(CARE_ENABLE_INLINE "Enable inlining of template functions" OFF)
```

Comment:
I think this should default to ON since I believe that will provide the behavior most people expect.